.travis.yml

```diff
@@ -9,3 +9,4 @@ env:
   global:
   - secure: qMfQvN4YQBktXvDuAO4JcZrJ4ES+UiDb3m5HY2HzOphyWE5i1SOhYG8eov5FHIZ5MqHDhDTrJcZYUs4Cp1G32sCnokeRbZ1AwKafK/6ZWW3nrN+3/bugPsK4sm4D1aYccVhBQdiMWIwU4yzCSlh8730CM8bGNb3eOFu0A5IhCGjO3v56sDtR26D82L+fEdSWwwoO/KTN2FKK+i3TbTjvOFSY9taJzl8yQkl86MYI4mGf3xDtYKlGMwnIw75Imr9YHFRLXQ3BK1zgadVAYg3U8aVZVIyuFDaGut3dNbjMTE/nreOyUDILyZ+HN8vnzbfdz9A6TQJO87jVBkJS42n0eoleXYiJL8C4VF7o5cOZV7V/JSF0IIn3+PBOG87gl9WRl4n0VSrLg27/Awqg9D4QEJexxqRKndK6OlDkgLOqtJwZDoS0JM/onDaNo2vZ6h9+qnY8d5H/940W6d90YNKQ5D0mwHintWuVu7mWzSY9g626NRM2zsfNA1brbbMdL00qcwPU+haGzmzDPPGdpSCgnMqSFsPAP59kbQ+tehEfxONboEhv1Rl5F2z6Jh2R1ZBNj0H+89dGX8O0UsRb0ZDY/adxfPECOWkwH5I4oPSBQTZLdPVgdsQwnP8pqrco944Cqc89nqJkQaY1Hu25gVweYr373t88qRa0lCg29ZSTuCo= #dcr DOCKER_USERNAME
   - secure: SntWiMDB2MKDBHYk0CvmKKX0YBhBGtH8Tvajv5nII8yA/QgKhdegmjv9sraP1CNrWxRiCN/cU2fs7LpFsVVoZuXTjc/ZqjI2W/taMs6N5S6HuJVnQ9b5Ts+YdT70IgvFRj8iE+/qVUe6VolKiYTGkikJGtMZxikqCOdYrOmoWXtixSn5VPs7hib3uuAfDwEQK4QMfM1r/AWXzqdHY7VwtAmX0PXQvKOJCe86umNtmDiu4RfRsxU9mVgrFPH1tHgLIVka89L4/XUWtpKUa7USvt6FUV4HoP2B5Igsc++UpJPjL0czQmEsZ94BiQSpz+OMSKyDDW5gkwLigIqZlgCdQMA5h5Rim/4b6hHuJdinZvR/0mww2BxIr0/kVFAx4t4jaTvm0GrDd4HEiuT7O7+Ep4rqNKPpi3+/Ru6QX9k0fX/qMhOpm/MoumQ0Cjtkh6uQpZCsbsg8JXezLA0cZLvIiN5DDzHbBMF4OAa2aZGjY5CPcrr3xWbbZOtJT5TCINgyNXnZZVa1HqKbZHLzbqJWJCPWvPkdHrGlAAUzQ1AmDUT/uhWtfZBrSu7iV55IL8leNVVag6dP/U5Tch+tmLbHn7YoE8fXP9QV70T2yVgn2v/rn1leYtYlQsdypkPxM+Efe6ZSz85pdpJKcMobjnIAXWQc0OPJsOJlQbp2/5Dhpwc= #dcr DOCKER_PASSWORD
+  - secure: "NJYgBr3FwGL5QOqWrobNyByPr+HXT3NgTWciHReL26gviO4mgLGoLU+IlqmqMLbEwogoC7dFGjIU8dm+R8UopQ0Zi6TEErYboQ94sN99HPbtpdPyEMJTbQc02q4APF7QGsX6AySfqkhJuL6KtK7B5Qwv/AeylQohProTKW2vPjyF1ZsKQGZ4agY7piSxp8fRufQYxkTc/ekcV4OcrjrbvCtM2gY+DWG7RMik/HIX4pBapS/LF0dC3XRpOOxruFB7OH9f++TbjhQLzJm/qzr20t0nxO0gB/jCrvgCjX3SIVe1eO20NsHaiJX7+PLjStIJ9eK8U1Agm0yBicjvzdbel+HhG6AqthYFJe1Sjd4o1FLwpsjI4geiqCpFqgEPPWfMEjL9FGYOpVqvlA072BsO22Hs9rBO2OzjytXevItD6CWC5wC5qQx8afOlgJJfsTcrj2r3iITj2W3LDeuSdR0sVqb4gHT8r6Br0K2EXY7pDe9plfdIdHkbtwf6prSAIUKzmjoFIYL5qzNEVQioOmsQ0SDHFEmPJtim/zS7cXAG8hvte19LciMePa8TxXvKpTl+3FQh29iPHoTLLQvTw9Aml8LTRPeqmTt1uN1Z14by/VXNQ1pafv9O6QzUId/gU7I49OHMp1XD6GPvwdyXCIS2AGXrJOMzdgFj3214K7lFYZU=" #COVERALLS_TOKEN
```

Comment:
we should agree how to store secrets...

Reply:
if we share accounts, probably yes! But some services are providing tokens "per repo", so probably if you create an account in coveralls and you allow them to read your github profile, probably you will see the same token for this project. In any case, if i ever die, you have permissions to override these vars with your credentials (that do have access to these repositories) :) . We can discuss, of course... but in this project, i don't think sharing credentials is required